src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+  const price = 40 * days;
+  const longVacation = 7;
+  const shotVacation = 3;
+
+  if (days >= longVacation) {
+    finalPrice = price - 50;
```

Comment:
Avoid using literals directly in expressions. Consider defining a constant for the discount amount, such as `LONG_TERM_DISCOUNT = 50`.